feedsx/src/main/java/com/likeminds/feedsx/post/create/view/LMFeedCreatePostFragment.kt

```diff
@@ -210,7 +210,13 @@ class LMFeedCreatePostFragment :
     // initializes the view as per the selected post type
     private fun initView() {
         binding.apply {
-            createPostExtras.attachmentUri?.let { selectedMediaUris.add(it) }
+            if (createPostExtras.attachmentType == com.likeminds.feedsx.posttypes.model.VIDEO) {
+                initiateMediaPicker(listOf(com.likeminds.feedsx.media.model.VIDEO))
+            }
+
+            if (createPostExtras.attachmentType == DOCUMENT) {
+                initiateMediaPicker(listOf(PDF))
+            }
```

Comment:
add proper checks, regarding selected media and convert boths `if` to `if-else if`